src/main/java/wiresegal/wob/TokenHolder.kt

```diff
@@ -32,6 +33,8 @@ val wikiIconUrl = System.getenv("WIKI_ICON") ?: "https://cdn.discordapp.com/emoj
 val embedColor = Color((System.getenv("ARCANUM_COLOR") ?: "003A52").toInt(16))
 val wikiEmbedColor = Color((System.getenv("WIKI_COLOR") ?: "CB6D51").toInt(16))
 
+const val progressCachePersistence: Long = 10 * 60 * 1000 // 10 minutes persistence for the cache
```

Comment:
Could use a long literal (i.e. `10L * 60 * 1000`) to give the compiler a hint here. Then the type ascription can go.